ev-rvs/nodejs-server-exercise
# Feature/evobe 79: write docker-compose file

### Ticket
https://phunts.atlassian.net/browse/EVOBE-79

### Changes Made
- Created `.env.example` file
- Created `Dockerfile` file
- Created `docker-compose.yml` file

### Notes
*N/A*

### Steps to Test
1. Clone this repo
2. Copy `.env.example` as `.env` and update `.env` file with `postgres` variables
3. Run `cd node-server-exercise`
4. Run ` docker-compose up --build` with `-d` optional as a daemon or without if you want to see container logs
